CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [Unreleased]
+
+### Added
+- Add a `SignalClientEvent` check in `SubscribeAndReceiveSubscribeAckTask` to immediately handles the failed task when WebSocket connection is terminated.
```

Comment:
nit: Spell check handles -> handle.
We do not have `SignalClientEvent` but `SignalingClientEvent`, checked [here](https://github.com/aws/amazon-chime-sdk-js/blob/5bb75405e2f3346ebbb8e39dc4d4bbdd1d2f4d3a/src/signalingclient/SignalingClientEvent.ts#L11).